docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -902,3 +902,29 @@ Once this is done, restart pfmon on all nodes using:
 
 NOTE: It is normal that pfmon will refuse to start on nodes B and C. Nonetheless you should execute the command on them to make sure you catch any stale process.
 
+MariaDB Galera cluster troubleshooting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Max connections reached
++++++++++++++++++++++++
+
+In the event that one of the 3 servers reaches the maximum amount of connections (defaults to 1000), this will dead-lock the Galera cluster synchronization. In order to resolve this, you should first increase database_advanced.max_connections, then stop packetfence-mariadb on all 3 servers, and follow the steps in the section 'Recovering from a split brain' of this document using anyone of the 3 servers as your source of truth.
```

Comment:
Should we briefly mention how to identify the source of truth? Or point the user in an other section from our doc that says how?